### ISSUE TITLE
Trouble installing deepchem on Sagemaker

### ISSUE THREAD

**Issue by coolotter** [deepchem/deepchem#1852]
Hi, 
I tried running deepchem on sagemaker and followed this tutorial for mu setup: https://aws.amazon.com/blogs/machine-learning/using-deepchem-with-amazon-sagemaker-for-virtual-screening/

I got this error when I tried testing whether the installation has worked properly:
![error_deepchem_notebook](https://user-images.githubusercontent.com/65193633/81958957-c04f7d00-95dc-11ea-8b4e-1e22df37e0b2.png)

This is the error that I got from the terminal"
![error_terminal](https://user-images.githubusercontent.com/65193633/81959098-f7259300-95dc-11ea-858f-a926b444c091.png)

Any ideas how to resolve this?

Thanks!

**Comment by rbharath** [comment]
@coolotter Oh this is really cool, I didn't know there was an article showing how to install DeepChem on sagemaker... 

This tutorial suggests installing from HEAD. It might be easier to install the last stable release

```
conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.3.0
```

I think what's happening in this particular error is that `jaccard_similarity_score` was removed in sklearn 0.23. Can you check your sklearn version? You might need to downgrade to sklearn 0.22 on your jupyterlab instance. I'll try to get a fix up for this in HEAD shortly so the amazon blog post directions work

**Comment by coolotter** [comment]
Thanks for the quick response! 
It says 0.20.3, I changed the version to 0.22 but now it says "ModuleNotFoundError: No module named 'rdkit'": 

![rdkit_error](https://user-images.githubusercontent.com/65193633/81973270-e59ab600-95f1-11ea-9742-367f14e98ac8.png)

Any ideas what to do? Will google colab work better? 

Thank you!

**Comment by rbharath** [comment]
Hmm, I think `rdkit` wasn't installed. Can you try running 

```
conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.3.0
```

I'll try running it on sagemaker myself once I get a chance in a week or two. For now, Colab is better tested :). Check out these directions:

https://forum.deepchem.io/t/getting-deepchem-running-in-colab/81

**Comment by coolotter** [comment]
Tried running that in AWS and did not work unfortunately. Google colab works perfectly fine! Though there was another issue that I encountered. 

Might be unrelated....but I am trying to use LIME, I kept getting error, and tried using this tutorial: https://deepchem.io/docs/notebooks/Explaining_Tox21.html
![lime_error](https://user-images.githubusercontent.com/65193633/82354277-9e843a80-99ce-11ea-8696-ab17b514ef45.png)

This is the shape of train_dataset.X and this is how I loaded my data:
Tried shap too earlier but it says it won't work on the disk dataset
![loading_dataset](https://user-images.githubusercontent.com/65193633/82354382-b956af00-99ce-11ea-9355-76a994fc0711.png)

Any ideas how to solve this?
